### DISCUSSION TITLE
where does this save the rbxl?

### DISCUSSION THREAD

**Discussion by aitherka** [luau/UniversalSynSaveInstance#37] [Q&A] [answered]
hello, i used this script to decompile a game on my executor but when i decompile it i do not know where its stored because theres no workspace folder, please help me

**Comment by aitherka** [comment]
and also, i use celery if this helps

**Comment by phoriah** [comment, marked as answer]
%temp%/celery

**Reply by aitherka** [reply]
thanks so much!
